src/algebraic_geometry/prime_spectrum/basic.lean

```diff
@@ -5,10 +5,10 @@ Authors: Johan Commelin
 -/
 import algebra.punit_instances
 import linear_algebra.finsupp
-import ring_theory.nilpotent
-import ring_theory.localization.away
-import ring_theory.ideal.prod
 import ring_theory.ideal.over
+import ring_theory.ideal.prod
+import ring_theory.localization.away
+import ring_theory.nilpotent
```

Comment:
Can you elaborate on the reasons behind the changes in this file? I don't really see the necessity and it messes up git blame.

Reply:
I can undo the changes; it was just some tidying up from a previous PR that I thought can be incorporated here.